source/EduCATS/Pages/Parental/FindGroup/Views/FindGroupPageView.cs

```diff
@@ -0,0 +1,203 @@
+
+		static Thickness _groupNumberFormPadding = new Thickness(20, 0);
+		static Thickness _baseSpacing = new Thickness(0, 10, 0, 0);
+		static Thickness _iosSettingsMargin = new Thickness(20, 60);
+		static Thickness _androidSettingsMargin = new Thickness(30);
+
+		public FindGroupPageView()
+		{
+			NavigationPage.SetHasNavigationBar(this, false);
+			BindingContext = new FindGroupPageViewModel(new PlatformServices());
+			createViews();
+		}
+
+		protected override void OnAppearing()
+		{
+			base.OnAppearing();
+		}
+
+		void createViews()
+		{
+			var backgroundImage = createBackgroundImage();
+			var settingsIcon = createSettingsIcon();
+			var mainLayout = createDataForm();
+
+			var scrollView = new ScrollView
+			{
+				VerticalOptions = LayoutOptions.FillAndExpand,
+				Content = new StackLayout
+				{
+					Children = {
+						mainLayout
+					}
+				}
+			};
+
+			Content = new Grid
+			{
+				HorizontalOptions = LayoutOptions.EndAndExpand,
+				Children = {
+					backgroundImage,
+					scrollView,
+					settingsIcon
+				}
+			};
+		}
+
+		StackLayout createDataForm()
+		{
+			var entryStyle = getEntryStyle();
+			var groupNumberEntry = createGroupNumberEntry(entryStyle);
+			var fIOEntry = createFIOEntry(entryStyle);
+			var findButton = createFindButton();
+
+			var mainStackLayout = new StackLayout
+			{
+				Spacing = _groupNumberFormSpacing,
+				Padding = _groupNumberFormPadding,
+				VerticalOptions = LayoutOptions.CenterAndExpand,
+				Children = {
+					groupNumberEntry,
+					fIOEntry,
+					findButton, 
+				}
+			};
+
+			mainStackLayout.SetBinding(IsEnabledProperty, "IsLoadingCompleted");
+			return mainStackLayout;
+		}
+
+		CachedImage createBackgroundImage()
+		{
+			return new CachedImage
+			{
+				Aspect = Aspect.AspectFill,
+				Source = ImageSource.FromFile(getRandomBackgroundImage())
+			};
+		}
+
+		CachedImage createSettingsIcon()
+		{
+			var settingsIcon = new CachedImage
+			{
+				HorizontalOptions = LayoutOptions.EndAndExpand,
+				VerticalOptions = LayoutOptions.StartAndExpand,
+				Margin = Device.RuntimePlatform == Device.iOS ? _iosSettingsMargin : _androidSettingsMargin,
+				Source = ImageSource.FromFile(Theme.Current.MainSettingsIcon),
+				Aspect = Aspect.AspectFill,
+				HeightRequest = _settingsIconSize,
+				WidthRequest = _settingsIconSize,
+				Transformations = new List<FFImageLoading.Work.ITransformation> {
+					new TintTransformation {
+						EnableSolidColor = true,
+						HexColor = Theme.Current.LoginSettingsColor
+					}
+				}
+			};
+
+			var tapGestureRecognizer = new TapGestureRecognizer();
+			tapGestureRecognizer.SetBinding(TapGestureRecognizer.CommandProperty, "SettingsCommand");
+			settingsIcon.GestureRecognizers.Add(tapGestureRecognizer);
+			return settingsIcon;
+		}
+
+		Entry createGroupNumberEntry(Style style)
+		{
+			var username = new Entry
+			{
+				Style = style,
+				ReturnType = ReturnType.Next,
+				Placeholder = "Введите номер группы"
```

Comment:
Should be a constant (language based). 
E.g. `CrossLocalization.Translate("parental_control_button_text")`
[Link to the languages](https://github.com/ilyalehchylin/educats-xamarin/tree/develop/source/EduCATS/Localization)